t/022-redirect.t

```diff
@@ -318,5 +316,4 @@ Location: http://agentzh.org/foo?a=b&c=d
 GET /read
 --- response_headers
 Location: http://agentzh.org/foo?a=b&c=d
---- response_body_like: 308 Permanent Redirect
```

Comment:
But it looks like nginx < 1.13.0 does not support "308 Permanent Redirect" code.